src/core/core/address.cpp

```diff
@@ -57,15 +57,16 @@
 
 #include <dsn/utility/ports.h>
 #include <dsn/utility/fixed_size_buffer_pool.h>
-#include <dsn/service_api_c.h>
-#include <dsn/cpp/address.h>
+
+#include <dsn/c/api_utilities.h>
+
+#include <dsn/tool-api/rpc_address.h>
+#include <dsn/tool-api/uri_address.h>
+#include <dsn/tool-api/group_address.h>
 #include <dsn/tool-api/task.h>
-#include "group_address.h"
-#include "uri_address.h"
 
 namespace dsn {
-const rpc_address rpc_group_address::_invalid;
-}
+const rpc_address rpc_address::sInvalid;
 
 #ifdef _WIN32
```

Comment:
We should no longer support win32. See L71, L247, L256 and L36 of this file, L81 of src/core/tests/address.cpp.